index.bs

```diff
@@ -6145,6 +6282,358 @@ However, [=authenticators=] that do not utilize [[!FIDO-CTAP]] do not necessaril
 :: [=largeblob|This extension=] directs the user-agent to cause the large blob to be stored on, or retrieved from, the authenticator. It thus does not specify any direct authenticator interaction for [=[RPS]=].
 
 
+## Device-bound public key extension (<dfn>devicePubKey</dfn>) ## {#sctn-device-publickey-extension}
```

Comment:
A minor editorial request: this header level and below needs an extra `#`. #1762 broke up extensions into two different groups and I'm hoping we can avoid another issue like #1781 to fix this afterwards.

Reply:
Done (by emlun).